models/stdp_dopamine_synapse.cpp

```diff
@@ -79,16 +74,23 @@ STDPDopaCommonProperties::set_status( const DictionaryDatum& d, ConnectorModel&
 {
   CommonSynapseProperties::set_status( d, cm );
 
-  long vtnode_id;
-  if ( updateValue< long >( d, names::vt, vtnode_id ) )
+  NodeCollectionDatum vt_datum;
+  if ( updateValue< NodeCollectionDatum >( d, names::volume_transmitter, vt_datum ) )
   {
+    if ( vt_datum->size() != 1 )
+    {
+      throw BadProperty( "Property volume_transmitter must be a single element NodeCollection" );
+    }
+
     const size_t tid = kernel().vp_manager.get_thread_id();
-    Node* vt = kernel().node_manager.get_node_or_proxy( vtnode_id, tid );
-    vt_ = dynamic_cast< volume_transmitter* >( vt );
-    if ( not vt_ )
+    Node* vt_node = kernel().node_manager.get_node_or_proxy( ( *vt_datum )[ 0 ], tid );
+    volume_transmitter* vt = dynamic_cast< volume_transmitter* >( vt_node );
+    if ( not vt )
     {
-      throw BadProperty( "Dopamine source must be volume transmitter" );
+      throw BadProperty( "Property vt must be set to a node of type volume_transmitter" );
```

Comment:
The property is now called "volume_transmitter" instead of "vt".

Reply:
Fixed in 0818dc83e.